linx-server.service

```diff
@@ -5,7 +5,10 @@ After=network.target
 [Service]
 User=linx
 Group=linx
-ExecStart=/usr/bin/linx-server -config /etc/webapps/linx-server/config.ini
+ExecStart=/usr/bin/linx-server -config /etc/linx-server/config.ini -allowMissingConfig
+WorkingDirectory=/var/lib/linx
+Restart=always
+RestartSec=2s
```

Comment:
Why exactly 2s? I guess the default is fine.

Reply:
The default is 0.01s, isn't it too often? I just took `always` and `2s` from gitea :)

Reply:
Alright, we can also keep it like this, 2s just seemd a little random to me :)